src/Akka.Streams.Kafka/Settings/ConsumerSettings.cs

```diff
@@ -69,8 +70,12 @@ public static ConsumerSettings<TKey, TValue> Create(Akka.Configuration.Config co
                 dispatcherId: config.GetString("use-dispatcher", "akka.kafka.default-dispatcher"),
                 autoCreateTopicsEnabled: config.GetBoolean("allow.auto.create.topics", true),
                 properties: properties,
-                connectionCheckerSettings: ConnectionCheckerSettings.Create(config.GetConfig(ConnectionCheckerSettings.ConfigPath)),
-                consumerFactory: null );
+                connectionCheckerSettings: ConnectionCheckerSettings.Create(
+                    config.GetConfig(ConnectionCheckerSettings.ConfigPath)),
+                consumerFactory: null)
+            {
+                VerboseLogging = config.GetBoolean("verbose-logging", false)
```

Comment:
Added a new setting - `VerboseLogging`. This is designed to eliminate `KafkaConsumerActor` spam.